config.yaml.sample

```diff
@@ -0,0 +1,13 @@
+# RENAME THIS FILE TO config.yaml ONCE CONFIGURED
+apprise:
+  summary: ON_ERROR
+  exceptions: True  # True or False (Whether to send or not exceptions)
+  promotions: True  # True or False (Whether to send or not incomplete promotions)

```

Comment:
To be a bit clearer, it'd be nice to change this to `apprise.notify.uncaught-exceptions` and `apprise.notify.incomplete-promotions`. Reads a bit better/is clearer IMO.